tests/test_vision_explanation.py

```diff
@@ -37,7 +38,7 @@ def test_vision_explain_preloaded():
     # save tested result in res
 
     # run the main function for saliency map generation
-    res = dr.get_drise_saliency_map(imgpath, None, None, None, savepath)
+    res = dr.get_drise_saliency_map(imgpath, None, None, savepath)
```

Comment:
Nit: may be good to have arg names here to describe the different params set to None.